src/closure.rs

```diff
@@ -135,10 +135,10 @@ mod tests {
 
     fn closure_fn(values: &[Value]) -> Option<Value> {
         assert_eq!(values.len(), 2);
-        let string: Option<String> = values[0].get();
-        assert_eq!(string, Some("test".to_string()));
-        let int: Option<i32> = values[1].get();
-        assert_eq!(int, Some(42));
+        let string_arg: String = values[0].get_some().unwrap();
```

Comment:
With `get_some()` changed, this would also then still have `Option<String>` as type here, or would need a second explicit `unwrap()` for the "None-ability" of the string.

I think it also makes various other code changes in this PR clearer